requirements.txt

```diff
@@ -5,3 +5,4 @@ djangorestframework==3.14.0
 Markdown==3.4.1
 pytz==2022.7
 sqlparse==0.4.3
+django-allauth==0.52.0
```

Comment:
This is not related to this PR. We can all Django all auto and set it up in other PR